### ISSUE TITLE
 When there are errors in the file code , why am I still getting +1 as code review in gerrit

### ISSUE THREAD

**Issue by karthikvee** [TouK/sputnik#142]
I am testing the following code in a file test.js with jshint like $jshint test.js:- 
`function myFunction(p1, p2) {
;
 ;
    return p1 == p2;              // The function returns the product of p1 and p2
}
`
and Iam getting following error

> test.js: line 2, col 1, Unnecessary semicolon.
test.js: line 3, col 2, Unnecessary semicolon.
2 errors

But while testing it using sputnik on gerrit, sputnik is giving +1 for it. When there are errors in the file , why am I still getting +1 as code review in gerrit

**Comment by pjagielski** [comment]
Hi, can you please attach Sputnik execution logs?

**Comment by karthikvee** [comment]
[output-sputnik-log.txt](https://github.com/TouK/sputnik/files/224280/output-sputnik-log.txt)

**Comment by pjagielski** [comment]
So in this log Sputnik complains about no jshint.json configuration, but there nothing regarding JSLint. Do you have `jslint.enabled=true` in `sputnik.properties`

**Comment by karthikvee** [comment]
@pjagielski No, jslint.enabled=false and Do I have to manually create jshint.json configuration, sputnik is supposed to take care of the configuration of PMD, Jshint etc because they are inbuilt right? or Do we have to manually configure them [PMD, Jshint, Checkstyle, Codenarc etc]

**Comment by SpOOnman** [comment]
@karthikvee did you manage to solve this configuration problem?
